client/my-sites/email/gsuite-add-users/index.jsx

```diff
@@ -71,8 +74,14 @@ class GSuiteAddUsers extends Component {
 	handleContinue = () => {
 		const { domains, productType, selectedSite } = this.props;
 		const { users } = this.state;
+		const validatedUsers = validateUsers( users );
 		const canContinue = areAllUsersValid( users );
```

Comment:
Should this not be running against `validatedUsers` instead of `users`? This could result in `canContinue` being incorrect and us allowing the user to move forward.
```suggestion
		const canContinue = areAllUsersValid( validatedUsers );
```